extensions/amp-consent/0.1/amp-consent.css

```
@@ -118,12 +118,29 @@ amp-consent.i-amphtml-consent-ui-iframe-active {
   transform: translate3d(0px, 100vh, 0px) !important;
 }
 
-amp-consent.i-amphtml-consent-ui-iframe-active.i-amphtml-consent-ui-enable-border {
+amp-consent.i-amphtml-consent-ui-iframe-active.i-amphtml-consent-modal {
+  box-sizing: border-box;
+  top: 0;
+  right: 0;
+  animation: 0.5s i-amphtml-modal-appear;
+  animation-fill-mode: forwards;
+  height: var(--modal-height) !important;
+  width: 90vw !important;
+  margin: auto !important;
+  max-width: 760px;
+}
+
+amp-consent.i-amphtml-consent-ui-border-enabled {
   border-top-left-radius: 8px !important;
   border-top-right-radius: 8px !important;
   box-shadow: 0 0 5px 0 rgba(0,0,0, 0.2) !important;
 }
 
+amp-consent.i-amphtml-consent-ui-border-enabled-modal {
```

Use compound classname, and see suggestion below:

```suggestion
amp-consent.i-amphtml-consent-ui-modal.i-amphtml-consent-ui-border-enabled {
```

---

Ahh, I see. I was a little confused on the subject of compound selectors, but this clears it up. Thanks!